src/main/java/frc/robot/Field.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.wpilibj.Filesystem;
+
+public class Field {
+
+    // Wrapper for loading an AprilTagFieldLayout from a JSON file
+    public static AprilTagFieldLayout getApriltagLayout() {
```

Comment:
We should be able to just use `AprilTagFields.k2023ChargedUp.loadAprilTagLayoutField()` for this

Reply:
Never mind, WPIlib hasn't release 2023.3.1 yet...